README.md

```diff
@@ -102,16 +99,11 @@ select * from db1.table1;
 
 Regression tests can be run in a local environment or in a Docker environment.
 
-To run regression tests locally, you need to have a Polaris server running locally, with the 
-`test` Authenticator enabled. You can do this by running Polaris as below:
-
-```shell
-java -Dquarkus.profile=test -jar quarkus/server/build/quarkus-app/quarkus-run.jar
-```
-
-Then, you can run the regression tests using the following command:
+To run regression tests locally, you first need to start Polaris, with default realm `POLARIS` and
+root credentials: `root:secret`, then run the tests:
```

Comment:
I guess we don't have to mention this as this code enforces it already.
```
tasks.named<QuarkusRun>("quarkusRun") {
  jvmArgs = listOf("-Dpolaris.bootstrap.credentials=POLARIS,root,secret")
}
```

Reply:
You are right, I removed the unnecessary part.